com.zsmartsystems.zigbee.dongle.ember/src/main/java/com/zsmartsystems/zigbee/dongle/ember/ezsp/EzspFrame.java

```diff
@@ -496,35 +496,12 @@ public static EzspFrameResponse createHandler(int ezspVersion, int[] data) {
         return null;
     }
 
-    /**
-     * Set the EZSP version to use
-     *
-     * @param ezspVersion the EZSP protocol version
-     * @return true if the version is supported
-     */
-    public static boolean setEzspVersion(int ezspVersion) {
```

Comment:
@charbopassman @sraillard Point tres important: on a eu des soucis avec cette partie la. J'ai verifie le repo officiel, il utilisent toujours la version EZSP sous forme de propriete statique. Le changement que j'avais fais justement est d'utiliser une propriete au niveau instance de sorte a pouvoir supporter des NCPs avec differentes versions.

Reply:
de mémoire, sans cette modification, ça plante, en effet on utilise la librairie en multi instances, ce que les autres utilisateurs ne font pas

Reply:
@mikomarrache j'ai bien en tête ce point, vous m'en avez déjà parlé.
Est-ce que ce commentaire appel un changement sur la PR? Pour moi dans notre master actuel cette méthode n'existe pas, il est donc cohérent de la supprimer.

Reply:
@charbopassman J'ai mis ce commentaire pour 2 raisons:

1. Noter l'importance de ce changement pour passenergy
2. A cause du fait que j'ai vu que la PR propose de supprimer les methodes statiques alors qu'elle n'existe plus dans master, du coup je n'ai pas trop compris pourquoi ca apparait dans la PR mais c'est probablement du a mon manque d'experience par rapport a GIT

Reply:
ok merci.
pour le point 1 c'et bien noté.
pour le point 2, cela montre que l'on supprime ces méthodes chez nous alors qu'elles sont dans la lib officielle. A priori cela va apparaitre à chaque PR.